src/common/schedulers/slurm_commands.py

```diff
@@ -293,7 +294,7 @@ def set_nodes_drain(nodes, reason):
 
 def set_nodes_power_down(nodes, reason=None):
```

Comment:
Add `@retry(stop_max_attempt_number=3, wait_fixed=1500)` on top of this as in develop